gnoi/cert/client.go

```diff
@@ -42,30 +43,33 @@ func (c *Client) Rotate(ctx context.Context, certID string, minKeySize uint32, p
 	if err != nil {
 		return fmt.Errorf("failed stream: %v", err)
 	}
-	if err = stream.Send(&pb.RotateCertificateRequest{
+	request := &pb.RotateCertificateRequest{
```

Comment:
You started a good thing here =)
would you be so kind as to rename all request variables as requests and response messages as response? But only in the code you are touching here. Thanks!

Reply:
No problem, will do